spec/features/guide_history_spec.rb

```diff
@@ -59,3 +63,23 @@ def events
       .reverse
   end
 end
+
+
+RSpec.describe "Guide history", type: :feature do
+  scenario "viewing previous editions" do
+    guide = create(:published_guide)
+    guide.editions << build(:edition, version: 2)
+
+    visit edition_comments_path(guide.reload.latest_edition)
+
+    expect_edition_to_be_open("Version #2")
+
+    click_link "Version #1"
+
+    expect_edition_to_be_open("Version #1")
+  end
+
+  def expect_edition_to_be_open(title)
+    expect(page).to have_css(".open-edition .panel-heading", text: title)
```

Comment:
The selector is checking the title but it's scoped to be nested under `.open-edition`.. so I suppose my assertion is saying.. "If this title heading can be found in an `open-edition` block then consider it to be open". But they could all be open right? My test doesn't say that the previous one is closed. I'll add that - thanks.

Do you feel strongly about not using the `open-edition` to indicate open-ness? Like I was saying above, I find it to be a good compromise.. Perhaps an alternative would be to check if the bootstrap class `.panel-body` is present? It looks like collapsed ones don't have one. Makes sense to me.. A collapsed thing doesn't have a body. Another one could be that it contains some nested `.event` divs? A collapsed thing shouldn't contain any events. What do you think?